README.md

```diff
@@ -97,6 +100,18 @@ modulizer owner/repo owner2/repo2
 
 This will create a `modulizer_workspace` directory and checkout the repos and their Bower dependencies and convert them all in place. You can then run `polymer serve` in the workspace directory and try out the results in Chrome 61 or Safari 10.1 (or Edge and Firefox with the appropriate flags turned on).
 
+
+## Conversion Options
+
+#### `--import-style ["name"|"path"]`
+
+Setting the import style allows you to set whether dependency imports are specifiered by npm package name, or relative file path. Importing specifiers that use package names are easier for third-party packages to work with, but unlike paths they currently can not run natively on the web. Defaults to "path".
+
+#### `--add-import-path`
+
+If included, the static `importPath` property will be added to converted Polymer elements. See [the `importPath` documentation](https://www.polymer-project.org/2.0/docs/devguide/dom-template) for more information.
+
```

Comment:
Add TODO (HTML comment) for other options

Reply:
These were the only two options that seemed to matter enough to be worth documenting, vs. just being helpful CLI flags for automating our own workflow. Maybe add `--flat` & `--private` as well?